website_baidu_map_multi/models/company_shop.py

```diff
@@ -0,0 +1,19 @@
+# 2016-2019 Elico Corp (https://www.elico-corp.com).
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class CompanyShop(models.Model):
+    _name = 'company.shop'
+
+    name = fields.Char(string='Name', required=True)
```

Comment:
remove useless strings (by default the name of the column is use if no string is specified)